skbio/alignment/_tabular_msa.py

```diff
@@ -1172,8 +1172,8 @@ def _slice_sequences_iloc_(self, i):
             return self._constructor_(new_seqs, positional_metadata=None)
         return self._constructor_(new_seqs)
 
-    def _get_sequence_loc_(self, l):
-        new_seqs = self._seqs.loc[l]
```

Comment:
glad that flake8 is picking up on this -- having `l` looks like `1`, and can be incredibly confusing.

Reply:
Aha yeah!